### PR TITLE
Add l10n it pec messages

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

**Comment by eLBati** [comment]
Travis is :white_check_mark:
Please review
@archetipo @angedras

**Comment by archetipo** [comment]
:+1:

File: l10n_it_pec_messages/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+
+Contributors
+------------
+
+ - Alessio Gerace <alessio.gerace@gmail.com>
+ - Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+ - Roberto Onnis <roberto.onnis@innoviu.com>
+""",
+    'images': [],
```
Comment (tafaRU) [inline l10n_it_pec_messages/__openerp__.py]: Please remove empty key

File: l10n_it_pec_messages/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        "view/ir_mail_server.xml",
+        "wizard/mail_compose_message_view.xml",
+        "view/mail_view.xml",
+        "view/res_users.xml",
+        "security/ir.model.access.csv",
+    ],
+    'demo': [
+        'demo/pec_data.xml',
+        ],
+    'test': [],
```
Comment (tafaRU) [inline l10n_it_pec_messages/__openerp__.py]: Same as above

File: l10n_it_pec_messages/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+##############################################################################
+
+{
+    "name": "Pec Messages",
+    "version": "1.0",
+    "author": "Agile Business Group",
```
Comment (tafaRU) [inline l10n_it_pec_messages/__openerp__.py]: Please add OCA as author.

File: l10n_it_pec_messages/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    "category": "Certified Mailing",
+    "website": "http://www.agilebg.com",
```
Comment (tafaRU) [inline l10n_it_pec_messages/__openerp__.py]: Please add missing key license.

**Comment by tafaRU** [comment]
Minor nitpickings, but technically :+1:
Please also consider to update copyright year in the header files, thanks!

**Comment by eLBati** [comment]
@tafaRU thanks, done

**Pull request merged**